src/components/qas/questions/qaMultiSelect/qaMultiSelect.module.scss

```diff
@@ -0,0 +1,69 @@
+:global {
+  @include react-md-menus-media;
+  @include react-md-selection-controls;
+
+  .qa-select {
+    label,
+    .label-pre,
+    .label-post {
+      @include copySecondary;
+      line-height: $baseLineHeight;
+    }
+
+    select {
+      @include copyPrimary;
+    }
+
+    &:not(.answerable) {
```

Comment:
this not answerable styling is working great, but please add a rule to make the text (like what would be the answer, but what starts out as the "Select" placeholder text) $neutral20 when .qa-select:not(.answerable).  This is easier to test and see if you temporarily make this multiselect question not the first question on the summary page

Reply:
added a `pointer-events: none;` as the fix for this

Reply:
👍